### PR TITLE
PIM-7806-7807: get one or several reference entity attribute options

### PR DESCRIPTION
**Pull request by NolwennP**

### CODE REVIEW

File: content/rest-api/resources.md [hunk 1]
```diff
@@ -1110,6 +1110,79 @@ Below is the JSON format representing a reference entity.
...
+  "validation_regexp": null
+}
+```
+
+::: panel-link Want more details about the reference entity attribute resource? [Check its endpoints here!](/api-reference.html#Referenceentityattribute)
+:::
+
+### Reference entity attribute option (3.x and EE only)
+
+Some type of attributes of reference entities can offer list of choices. These available choices are reference entity attribute options.
```
Comment (ahocquard) [inline content/rest-api/resources.md]:
```suggestion
Some type of attributes of reference entities can offer a list of choices. These available choices are reference entity attribute options.
```

not sure

File: content/rest-api/resources.md [hunk 2]
```diff
@@ -1110,6 +1110,79 @@ Below is the JSON format representing a reference entity.
...
+
+::: warning
+Note that "Reference entity attribute option" are not the same entity than "Attribute option", even though they have a lot in common. The first one is an option of an attribute used to describe reference entities records, the second one is an option of an attribute used to describe products.
+:::
+
+Only attribute of type simple select or multiselect can have options.
```
Comment (ahocquard) [inline content/rest-api/resources.md]: is it the same wording choosed by Delphine? I though it was different

File: content/swagger/resources/reference_entity_attributes/definitions/reference_entity_attribute.yaml [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    x-validation-rules: The `localeCode` is the code of an existing and activated locale
+    default: {}
+    properties:
+      localeCode:
+        type: string
+        description: Attribute label for the locale `localeCode`
+  type:
+    type: string
+    description: Attribute type
+    enum: ['text', 'image']
```
Comment (ahocquard) [inline content/swagger/resources/reference_entity_attributes/definitions/reference_entity_attribute.yaml]: record?

File: content/swagger/resources/reference_entity_attributes/definitions/reference_entity_attribute.yaml [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+  localizable:
```
Comment (ahocquard) [inline content/swagger/resources/reference_entity_attributes/definitions/reference_entity_attribute.yaml]: we are not very consistent. `is_textarea` but not `is_localizable`

File: content/rest-api/resources.md (hunk 2 above)
Comment (NolwennP) [reply content/rest-api/resources.md]: You are right, I forgot to update it here.

File: content/rest-api/resources.md [hunk 5]
```diff
@@ -1110,6 +1110,79 @@ Below is the JSON format representing a reference entity.
 ::: panel-link Want more details about the reference entity resource? [Check its endpoints here!](/api-reference.html#Referenceentity)
 :::
 
+### Reference entity attribute (3.x and EE only)
+
+Reference entities have attributes that allow to describe their [records](/documentation/resources.html#reference-entity-record-3x-and-ee-only).
+
+::: warning
+Note that "Reference entity attribute" are not the same entity than "Attribute", even though they have a lot in common. The first one is used to describe reference entities records, the second one is use to describe products.
```
Comment (tamarasaurus) [inline content/rest-api/resources.md]:
```suggestion
Note that "Reference entity attribute" are not the same entity as "Attribute", even though they have a lot in common. The first one is used to describe reference entities records, the second one is use to describe products.
```

**Pull request merged**